OWNERS_ALIASES

```diff
@@ -436,14 +414,10 @@ aliases:
   #   -
   #   -
   sig-contribex-approvers:
-    - alisondy
     - mrbobbytables
     - cblecker
     - nikhita
   sig-docs-approvers:
-    - jimangel
-    - kbhawkey
-    - onlydole
```

Comment:
these are current leads from https://github.com/kubernetes/community/tree/master/sig-docs ... seems weird to be dropping them

Reply:
Adding them back for now.

Reply:
FWIW - I think they can be removed if we remove docs dir which has been empty since 2019.